gamemodes/terrortown/gamemode/shared/hud_elements/base_elements/pure_skin_element.lua

```diff
@@ -62,7 +62,7 @@ if CLIENT then
 
 		-- draw text
 		if t then
-			draw.AdvancedText(t, "PureSkinBar", x + 14, y + 1, util.GetDefaultColor(c), TEXT_ALIGN_LEFT, TEXT_ALIGN_LEFT, true, s)
+			draw.AdvancedText(t, "PureSkinBar", x + 14, y + h/2 - 1, util.GetDefaultColor(c), TEXT_ALIGN_LEFT, TEXT_ALIGN_CENTER, true, s)
```

Comment:
please change `h / 2` to `0.5 * h` because that is faster in lua

Reply:
Good to know. I changed it. 
(I probably need to go through my other lua code as well.)